hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/batch/JsonMessageBatch.java

```diff
@@ -36,21 +39,16 @@
     private boolean closed = false;
     private int retryCounter = 0;
 
-    public JsonMessageBatch(String id, ByteBuffer buffer, int size, int batchTime, Clock clock, List<Header> additionalHeaders) {
+    JsonMessageBatch(String id, ByteBuffer buffer, Subscription subscription, Clock clock) {
```

Comment:
Why it's not `public` anymore?

Reply:
`public` in this case was never needed in fact

Reply:
backward compatibility?